packages/teams-js/test/public/authentication.spec.ts

```diff
@@ -51,12 +51,14 @@ describe('Testing authentication capability', () => {
     HostClientType.desktop,
     HostClientType.android,
     HostClientType.ios,
+    HostClientType.ipados,
```

Comment:
why are we adding this extra ipados?

Reply:
Please don't be blocked on my comment (I am logging off for the day). I was asking just out of curiosity.

Reply:
It got added to the authenticate function a few months ago but this unit test didn't get updated at the time.